### PR TITLE
added atma.io solution + user

### PR DESCRIPTION
**Pull request by gse-atma-dev**
This update introduces our solution 'atma.io', laying the groundwork for the future inclusion of conformance tests.

### CODE REVIEW

File: catalog/solutions/atma_io/2.0.1/solution.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "atma_io",
```
Comment (raimundo-henriques) [inline catalog/solutions/atma_io/2.0.1/solution.json:2]:
```suggestion
  "id": "atma-io",
```

File: catalog/users/atma_io.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "id": "atma_io",
```
Comment (raimundo-henriques) [inline catalog/users/atma_io.json:2]:
```suggestion
  "id": "atma-io",
```

File: catalog/solutions/atma_io/2.0.1/solution.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+  "name": "atma.io",
+  "version": "2.0.1",
+  "website": "https://atma.io/",
+  "provider": "atma_io",
```
Comment (raimundo-henriques) [inline catalog/solutions/atma_io/2.0.1/solution.json:6]:
```suggestion
  "provider": "atma-io",
```

**Review by raimundo-henriques — CHANGES_REQUESTED** [review]
@gse-atma-dev Thank you for your submission!

Both the `solution-id` and the `user-id` must contain only letters, numbers, and dashes. 

Can you please replace the underscores (`_`) by dashes (`-`)?

Please see the inline comments.

cc. @prasadt1 
Thank you,

File: catalog/users/atma_io.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+  "kind": "solutionprovider",
+  "name": "atma.io",
+  "email": "hello@atma.io",
+  "website": "https://www.atma.io/"
```
Comment (raimundo-henriques) [inline catalog/users/atma_io.json:6]: You might want to add the optional field

```
"logo": "<url-to-your-logo>"
```

That ensures that in your logo is displayed in the collaborators page.

I am sorry that the possibility of adding the "logo" field was not explicit in the documentation. I have updated it meanwhile.

File: catalog/users/atma-io.json [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{
+  "id": "atma-io",
+  "kind": "solutionprovider",
+  "name": "atma.io",
+  "email": "hello@atma.io",
+  "website": "https://www.atma.io/"
```
Comment (raimundo-henriques) [inline catalog/users/atma-io.json:6]: You might want to add the optional field
```
"logo": "<url-to-your-logo>"
```
That ensures that in your logo is displayed in the collaborators page.

I am sorry that the possibility of adding the "logo" field was not explicit in the documentation. I have updated it meanwhile.

**Review by raimundo-henriques — COMMENTED** [review]
@gse-atma-dev  Thank you for your submission!

We took the liberty to go ahead and fix some minor syntactical problems: since the use of underscores is not allowed by the catalog's syntax, we replaced them by dashes. 

Can you please review the changes?

Also, we strongly recommend adding a logo to your user `json`. Please see the inline comment.

Thank you,
Best,
cc @prasadt1

**Review by raimundo-henriques — COMMENTED** [review]
@gse-atma-dev  Thank you for your submission!

We took the liberty to go ahead and fix some minor syntactical problems: since the use of underscores is not allowed by the catalog's syntax, we replaced them by dashes. 

Can you please review the changes?

Also, we strongly recommend adding a logo to your user `json`. Please see the inline comment.

Thank you,
Best,
cc @prasadt1

**Comment by gse-atma-dev** [comment]
@raimundo-henriques Sorry for the delay and thank you for making the changes to the solution name.
As requested we added the logo and also added the results from the first conformance test with cirrus-nexus. 

Will add the result of the second compliance test (as soon as it finalized) via a second PR.

**Review by raimundo-henriques — CHANGES_REQUESTED** [review]
Thank you very much for the changes. There are, however, a couple of issues that still need fixing (but which are easy to fix):
- The `"logo"` field belongs in the `atma-io.json` file, rather than on the `solution.json`. Please see my inline suggestions on how to fix these;
- The conformance test result json is missing a comma, please see my inline suggestion;
- The version of the Pathfinder Tech Specs tested must be either `"1.0.1"` or `"2.0.0"`, please see my inline suggestions.
Thank you for your cooperation!

**Comment by gse-atma-dev** [comment]
hi, thanks for checking

I added your suggestions but the validation for the version in the pipeline seems now to fail again (I think it does not fail with version 2.0.1, see other PR from true carbon)

**Comment by gse-atma-dev** [comment]
ah thank you, yes I think if everything is good from your side I think we can merge it

Thanks

**Review by raimundo-henriques — APPROVED** [review]

**Comment by gse-atma-dev** [comment]
The Vercel deployment action is failing, do we have to resolve this before merging the PR? can you merge it or do we have to do this?

**Comment by raimundo-henriques** [comment]
Hello again @gse-atma-dev,
That's just because the `main` branch your branch is based on is already outdated, so Vercel's preview will break when trying to build it. This will be solved on merge, which will happen very soon!

Thank you!

**Pull request merged**